tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Remove blank line between comment and code</li><li>State the buffer size in the comment</li><li>Use this notation <code>1e6</code>  instead of <code>1000000</code><span class="ql-cursor">﻿</span></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>done</p>